main/main.go

```diff
@@ -108,7 +108,7 @@ func main() {
 		protocol:       &p,
 		env:            conf.Env,
 		authServiceURL: conf.Niomon,
-		MessageHandler: make(chan HTTPRequest, 100),
+		MessageHandler: make(chan HTTPRequest, 200),
```

Comment:
Why 200? Can we make this configurable?

Reply:
i added a comment and will note to make it configurable in a future release